### ISSUE TITLE
New Window after login.. Is there a way to find a root element(desktop)?

### ISSUE THREAD

**Issue by SharonJacob** [Roemer/FlaUI#55]
My application launches a new window after login which is not the child window.
If I have the root element, then I can find the child(new window) under the root element.

How can I find the root element?

The below code that I used to launch my application and login cannot find the new window.
            var app = Application.Launch("CC.exe");
            var automation = new UIA3Automation();
            var window = app.GetMainWindow(automation);

             //Code for login and wait for new window to launch..         

            var _ccMain= window.FindFirstDescendant(cf => cf.ByAutomationId("Shell"));

**Comment by Roemer** [comment]
The problem probably is that the new window is not a child of the login window (which is your MainWindow at startup). You should be able to get the new window with ```Application.GetAllTopLevelWindows(automation)``` and continue from there. Otherwise it would be helpful if you could post a screenshot of FlaUInspect, showing which elements you have and want.

**Comment by Roemer** [comment]
Oh and if you want to find the Desktop itself, you can use ```automation.GetDesktop()```

**Comment by SharonJacob** [comment]
Great automation.GetDesktop() helps

**Comment by alayor-westga** [comment]
@SharonJacob Could you share how you made it work with automation.GetDesktop()?

**Comment by RikiSharon** [comment]
> @SharonJacob Could you share how you made it work with automation.GetDesktop()?

I used `automation.GetDesktop()` for my needs, maybe it can help you.

I have a ComboBox in my app, but I could not get its Items. (Got NULL Exception).
In FlaUI Inspect, I saw the list is not a child of the ComboBox or the app, but is child of the Desktop, without any containing app. 
So I used this code:

`AutomationElement desktop = automation.GetDesktop();`
`AutomationElement list = desktop.FindFirstDescendant(f => f.ByName("My List"));`
